src/components/policy/policy_regular/src/policy_helper.cc

```diff
@@ -356,14 +354,29 @@ bool CheckAppPolicy::operator()(const AppPoliciesValueType& app_policy) {
                "Permissions for application:" << app_id
                                               << " have been changed.");
 
+  if (IsPredefinedApp(app_policy)) {
+    for (auto it = snapshot_->policy_table.app_policies_section.apps.begin();
```

Comment:
@AKalinich-Luxoft May be range based loop?

Reply:
@AByzhynar yes, in this case code little bit shorter :) Done in https://github.com/AKalinich-Luxoft/sdl_core/pull/1/commits/64b69a56b7aa1b6ece996a9c8f1332098a1f5e62